cypress/integration/networking.spec.ts

```diff
@@ -1,4 +1,4 @@
-import * as commands from "../support/commands";
+import * as cmd from "../support/commands";
```

Comment:
Can we stick with `commands`? We are using using the full word in all other tests and I’d prefer to be consistent and keep the changes to a minimum. (In general I also prefer to use the module name as the qualified import to aid readability at the sacrifice of writing code quicker.)

Reply:
Consistency muss sein: 3318e58b06e141e686e4fdb5853d624815179d1d.
Just wanted to avoid line wrapping in some places.